### PR TITLE
[tflitefile_tools] Introduce unittest

### PR DESCRIPTION
**Pull request by YongseopKim**
This commit introduces the basic unittest module.

Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

--

for https://github.com/Samsung/ONE/issues/8086

PTAL @hseok-oh and @chunseoklee

### CODE REVIEW

**Comment by YongseopKim** [comment]
```
~/Works/ONE/tools/tflitefile_tool(pr/introduce_unittest ✗) python -m unittest discover
....
----------------------------------------------------------------------
Ran 4 tests in 0.001s

OK
```

**Comment by YongseopKim** [comment]
I think I can add more tests after removing tflite dependency.

File: tools/tflitefile_tool/tests/test_setup.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os.path
+import unittest
+
+# Python doesn't have const var but handle these as const
+# It's meaning that DO NOT MODIFY these vars
+CACHED_TEST_MODEL_DIR = "../../tests/scripts/models/cache"
+TEST_MODEL_PATH = os.path.join(CACHED_TEST_MODEL_DIR, "convolution_test.tflite")
```
Comment (hseok-oh) [inline tools/tflitefile_tool/tests/test_setup.py:23]: If we want to use simple test model on host without download, we can use `nnpackage/examples/v1.0.0/add/add.tflite` file.

File: tools/tflitefile_tool/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+```
+ONE$ find ./tests/scripts/models/ -name "convolution_test.tflite"
+./tests/scripts/models/cache/convolution_test.tflite
+```
+
+## Test
+
+```
+ONE/tools/tflitefile_tool$ python -m unittest discover
```
[the remark this answers is not in the document]
Comment (YongseopKim) [reply tools/tflitefile_tool/tests/README.md:15]: AFAIU, yes the `discover` arg is necessary.

File: tools/tflitefile_tool/tests/test_setup.py (hunk 1 above)
Comment (YongseopKim) [reply tools/tflitefile_tool/tests/test_setup.py:23]: If so, I'll change. I just need a simple tflite model.

File: tools/tflitefile_tool/tests/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (YongseopKim) [reply tools/tflitefile_tool/tests/README.md:15]: To clarify, there are two options to run tests. But AFAIU `python -m unittest` doesn't work.

**Comment by YongseopKim** [comment]
```
~/Works/ONE/tools/tflitefile_tool(pr/introduce_unittest ✗) python -m unittest discover
....
----------------------------------------------------------------------
Ran 4 tests in 0.001s

OK
```

File: tools/tflitefile_tool/tests/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chunseoklee) [reply tools/tflitefile_tool/tests/README.md:15]: I will look into the unistest manual later :) Thank you

**Comment by YongseopKim** [comment]
PTAL @chunseoklee @hseok-oh again.

File: tools/tflitefile_tool/tests/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chunseoklee) [reply tools/tflitefile_tool/tests/README.md:15]: FYI. `python -m unittest` w/o discover works on my local machine. 

```
~/d/O/t/tflitefile_tool ⬢  8.10.0
➜  python -m unittest
....
----------------------------------------------------------------------
Ran 4 tests in 0.001s

OK

~/d/O/t/tflitefile_tool ⬢  8.10.0
➜  git rev-parse --short HEAD
c1acb2953

```

**Review by chunseoklee — APPROVED** [review]

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**